torch/_torch_docs.py

```diff
@@ -625,6 +625,10 @@ def merge_dicts(*dicts):
 Keyword args:
     {out}
 
+.. note::  From version 1.8 onwards, the angle function returns `PI` for negative real numbers,
```

Comment:
"Starting in PyTorch 1.8, angle returns pi for negative real numbers, zero for non-negative real numbers, and propagates NaNs. Previously the function would return zero for all real numbers and not propagate floating-point NaNs."